tests/views/command_line_tests.py

```diff
@@ -43,6 +46,9 @@ def setUpClass(cls):
             with captured_stdout():
                 management.call_command("packages", operation="load_package", source=test_pkg_path, yes=True, verbosity=0)
 
+        path_to_cheesy_image = Path(settings.MEDIA_ROOT) / "uploadedfiles" / "test.png"
+        cls.addClassCleanup(os.unlink, path_to_cheesy_image)
```

Comment:
That ended up exposing this file under a place tracked by git, so I cleaned it up.